commands/Music/queue.js

```diff
@@ -7,7 +7,6 @@ module.exports = class extends Command {
         super(...args, {
             runIn: ["text"],
             cooldown: 10,
-            aliases: ["stopmusic", "leave"],
```

Comment:
Why were these aliases removed?

Reply:
You want to trigger stopmusic and leave commands on the queue command? The user shouldn't stop playing music when they fetch the queue.

Reply:
Lmfao, you're not wrong. mb